### DISCUSSION TITLE
how to set time_limit in highs in python

### DISCUSSION THREAD

**Discussion by yokawhhh** [ERGO-Code/HiGHS#2071] [Q&A] [answered]
It seems setOptionValue can do this. But no direct document can describe how to use it. And other controlling parameters seems also not complete in python.

**Comment by jajhall** [comment, marked as answer]
The options are documented in https://ergo-code.github.io/HiGHS/stable/options/definitions/

In Python (and C++) you set the time limto to 3600s thus

> h.setOptionValue("time_limit", 3600);
